kie-server-parent/kie-server-controller/kie-server-controller-websocket/src/main/java/org/kie/server/controller/websocket/client/WebSocketKieServerClient.java

```diff
@@ -94,258 +98,302 @@ public WebsocketKieServerClient(String url) {
                 
                 @Override
                 public void unregisterQuery(String queryName) {
-                    throw new UnsupportedOperationException("Not supported for websocket implementation");
+                    throwUnsupportedException();
                 }
                 
                 @Override
                 public void setResponseHandler(ResponseHandler responseHandler) {
-                    throw new UnsupportedOperationException("Not supported for websocket implementation");
+                    throwUnsupportedException();
                 }
                 
                 @Override
                 public void replaceQuery(QueryDefinition queryDefinition) {
-                    CommandScript script = new CommandScript(Collections.singletonList((KieServerCommand) new DescriptorCommand("QueryDataService", "replaceQuery",  
-                            WebsocketUtils.marshal(MarshallingFormat.JSON.toString(), queryDefinition), MarshallingFormat.JSON.toString(), new Object[]{queryDefinition.getName()})));
-                    sendCommandToAllSessions(script, new WebsocketServiceResponse(true, (message) -> {
-                        WebsocketUtils.unmarshal(message, MarshallingFormat.JSON.getType(), ServiceResponsesList.class);            
+                    CommandScript script = new CommandScript(Collections.singletonList((KieServerCommand) new DescriptorCommand("QueryDataService", "replaceQuery",
+                                                                                                                                WebSocketUtils.marshal(MarshallingFormat.JSON.toString(), queryDefinition), MarshallingFormat.JSON.toString(), new Object[]{queryDefinition.getName()})));
+                    sendCommandToAllSessions(script, new WebSocketServiceResponse(true, (message) -> {
+                        WebSocketUtils.unmarshal(message, MarshallingFormat.JSON.getType(), ServiceResponsesList.class);
                         return null;
                     })).getResponses();    
                 }
                 
                 @Override
                 public void registerQuery(QueryDefinition queryDefinition) {
-                    throw new UnsupportedOperationException("Not supported for websocket implementation");
+                    throwUnsupportedException();
                 }
                 
                 @Override
                 public <T> List<T> query(String queryName, String mapper, String builder, Map<String, Object> parameters, Integer page, Integer pageSize, Class<T> resultType) {
-                    throw new UnsupportedOperationException("Not supported for websocket implementation");
+                    throwUnsupportedException();
+                    return null;
```

Comment:
what's the point of returning null after throwing UnsupportedException, isn't this unreachable code

Reply:
only to make the compiler happy :) but definitely improved now.